components/script/dom/webidls/Doge.webidl

```diff
@@ -0,0 +1,15 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// https://jeenalee.github.io/doge-standard/
+
+typedef sequence<DOMString> DogeInit;
+
+[Constructor(optional DogeInit init),
```

Comment:
We now use a different constructor syntax that looks like [this](https://github.com/servo/servo/blob/59c68e2eb7afdda17fc27e866360a1746a78bbda/components/script/dom/webidls/AnalyserNode.webidl#L18) instead. You can see how the transition happened in https://github.com/servo/servo/commit/9ce82ea1aedfd3eddad0eed9fac6718599a25ec3 .